luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/LuthierIndustrialPark.java

```diff
@@ -37,29 +41,36 @@
     private final LuthierResourceDictionaries resourceDictionaries;
     private final FactoryPark<Dimension> dimensionFactoryPark;
     private final FactoryPark<SearchProvider> searchProviderFactoryPark;
+    private final FactoryPark<ConfigPhysicalTable> physicalTableFactoryPark;
+    private final DataSourceMetadataService metadataService;
 
     /**
      * Constructor.
      *
      * @param resourceDictionaries  The dictionaries to initialize the industrial park with
-     * @param dimensionFactories The map of factories for creating dimensions from external config
-     * @param searchProviderFactories The map of factories for creating dimensions from external config
+     * @param dimensionFactories  The map of factories for creating dimensions from external config
+     * @param searchProviderFactories  The map of factories for creating from external config
+     * @param physicalTableFactories  The map of factories for creating physicalTables from external config
      */
     protected LuthierIndustrialPark(
             LuthierResourceDictionaries resourceDictionaries,
             Map<String, Factory<Dimension>> dimensionFactories,
-            Map<String, Factory<SearchProvider>> searchProviderFactories
+            Map<String, Factory<SearchProvider>> searchProviderFactories,
+            Map<String, Factory<ConfigPhysicalTable>> physicalTableFactories
     ) {
         this.resourceDictionaries = resourceDictionaries;
         Supplier<ObjectNode> dimensionConfig = new ResourceNodeSupplier("DimensionConfig.json");
         Supplier<ObjectNode> searchProviderConfig = new ResourceNodeSupplier("SearchProviderConfig.json");
-        dimensionFactoryPark = new FactoryPark<>(dimensionConfig, dimensionFactories);
-        searchProviderFactoryPark = new FactoryPark<>(searchProviderConfig, searchProviderFactories);
+        Supplier<ObjectNode> physicalTableConfig = new ResourceNodeSupplier("PhysicalTableConfig.json");
+        // Supplier<ObjectNode> logicalTableConfig = new ResourceNodeSupplier("LogicalTableConfig.json");
+        this.dimensionFactoryPark = new FactoryPark<>(dimensionConfig, dimensionFactories);
+        this.searchProviderFactoryPark = new FactoryPark<>(searchProviderConfig, searchProviderFactories);
+        this.physicalTableFactoryPark = new FactoryPark<>(physicalTableConfig, physicalTableFactories);
+        this.metadataService = new DataSourceMetadataService();
```

Comment:
We should not be hardcoding this. I'm not fully up to speed on how the
metadataService is supposed to work, but if we're expecting to use a single
service for every physical table, then we should make this a constructor
parameter on the industrial park. If we expect different physical tables to
have different metadata services, well, then we need to give our customers the
ability to configure and attach them to their physical table configuration.

@mclawhor How is the metadataService supposed to work?

Reply:
After talking to @michael-mclawhorn, it looks like we're basically doing this as is already. So we don't have to worry about changing this yet.

Reply:
I am also not fully happy with the fact this is hard coded. 
but LIP is clearly the place that makes _most_ sense for the metadataService to live in, unless we supply user an entire different scope to configure that.